### DISCUSSION TITLE
Can the TOC *not* disappear with page resize?

### DISCUSSION THREAD

**Discussion by timdesrochers** [requarks/wiki#5765] [Help / Questions]
I have a lot of longer docs in which I rely on the auto-generated TOC to navigate. Especially useful when sharing which others, which is the whole point of my instance. The (apparent?) inability to either pin the TOC to the top of the page or after an introduction (ala Wikipedia) is extremely frustrating, especially since it does not appear at all on mobile, which is a huge amount of the traffic I need to serve.

**Comment by viharm** [comment]
I know this is an old thread, but I've found this unusual bug, where the page nav/sidebar disappears if the viewport is smaller than a predefined size. This is fantastic for responsive application, but as @timdesrochers  has pointed out negatively affects usability for the end-user. This becomes especially relevant since long docs will almost always have multiple headings.

Is this intentional? Is there a way to have a toc and tag matrix on smaller viewports?

PS: I've been comparing a lot of wiki software, and I really really like where wiki.js is headed. Congratulations to the devs for many of the design & engineering decisions! :clap:
